scipy/stats/_discrete_distns.py

```diff
@@ -336,9 +336,9 @@ def _cdf(self, x, n, p):
 
     def _logcdf(self, x, n, p):
         k = floor(x)
+        k, n, p = np.broadcast_arrays(k, n, p)
         cdf = self._cdf(k, n, p)
         cond = cdf > 0.5
-
```

Comment:
looks like white space was accidentally deleted

Reply:
Linter is not complaining, so we'll let it be.